src/track_vaccine.py

```diff
@@ -1,12 +1,85 @@
-from requests_html import HTMLSession
-import time
+import hashlib
+import json
 import os
+import sys
+import time
 
+from requests_html import HTMLSession
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+
+from src.email_alert import email
 from src.state_manager import StateManager
 
 TURBO_VAX_URL = "https://www.turbovax.info/"
 
+class VaccinationSite(object):
+    def __init__(self, name, neighborhood, numAvailable, time):
+        self.name = name
+        self.neighborhood = neighborhood
+        self.time = time # Formatted like Apr 8 - 1:15PM
+        self.numAvailable = numAvailable
+    def availability(self):
```

Comment:
lets just override the __str__ method, rather than use availability() method.

so str(site) returns the info we need.

def __str__(self):
    return 'string representation'